policy/modules/services/resolvconf.te

```diff
@@ -0,0 +1,36 @@
+policy_module(resolvconf, 1.0)
+
+########################################
+#
+# Declarations
+#
+
+attribute_role resolvconf_roles;
+
+type resolvconf_t;
+type resolvconf_exec_t;
+init_system_domain(resolvconf_t, resolvconf_exec_t)
+role resolvconf_roles types resolvconf_t;
+
+########################################
+#
+# Local policy
+#
+corecmd_exec_bin(resolvconf_t)
+corecmd_exec_shell(resolvconf_t)
+
+## shell script permissions
```

Comment:
Using `##` triggers a warning in `make conf`, when `python3 -t -t -E -W error support/segenxml.py -w -T tmp/iftemplates -m policy/modules/services/resolvconf` is run:
```
support/segenxml.py: warning: orphan XML comments at bottom of file policy/modules/services/resolvconf.te
```

Please use `#` instead, here and for `## special permissions`.